geomesa-ui/src/main/ui/src/app/card/card.js

```diff
@@ -41,4 +41,16 @@ angular.module('geomesa.card', [])
                 }, true);
             }
         };
-    }]);
+    }])
+
+    .directive('drag', function(){
+        return {
+
```

Comment:
should we restrict to 'A' (attribute), wouldn't really make sense to have this directive as an element